app/src/androidTest/java/com/github/se/travelpouch/ui/home/StorageDashboardTests.kt

```diff
@@ -0,0 +1,54 @@
+package com.github.se.travelpouch.ui.home
+
+import androidx.activity.compose.LocalActivityResultRegistryOwner
+import androidx.activity.result.ActivityResultRegistry
+import androidx.activity.result.ActivityResultRegistryOwner
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.times
+
+class StorageDashboardTests {
+  @Composable
+  fun withActivityResultRegistry(
+      activityResultRegistry: ActivityResultRegistry,
+      content: @Composable () -> Unit
+  ) {
+    val activityResultRegistryOwner =
+        object : ActivityResultRegistryOwner {
+          override val activityResultRegistry = activityResultRegistry
+        }
+    CompositionLocalProvider(
+        LocalActivityResultRegistryOwner provides activityResultRegistryOwner) {
+          content()
+        }
+  }
+
+  private lateinit var navigationActions: NavigationActions
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Before
+  fun setUp() {
+    navigationActions = mock(NavigationActions::class.java)
+  }
+
+  @Test
+  fun everythingIsDisplayed() {
+    composeTestRule.setContent {
+      withActivityResultRegistry(mock(ActivityResultRegistry::class.java)) {
+        StorageDashboard(navigationActions)
+      }
+    }
+
+    composeTestRule.onNodeWithTag("StorageDashboard").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("StorageBar").assertIsDisplayed()
```

Comment:
Your test seems a bit light. You should test that the StorageBar, the goBackButton, the card for the storage limit and so on are displayed.